_posts/2019_NIPS_speakers/2019-10-21-Template.md

```diff
@@ -0,0 +1,12 @@
+---
+name: 
+title: 
+modal-id: 1
+img: robot.png      
+alt: Picture of
+topic: 
+bio: 
+website: 
+tags: 
```

Comment:
Here is where you add the object **tag** reference 'oral-neurips2019'

Reply:
I'm not sure I do understand this comments. But since I have no information for the oral and keynote speakers we can wait to do this changes. And push just the current changes with schedule, Program Committee and poster presenters